appstore/appstore/_version.py

```diff
@@ -1 +1 @@
-__version__ = "1.3.1-alpha"
+__version__ = "1.4.0-dev"
```

Comment:
Should this be a non-dev version?

Reply:
Yes, this should have been updated manually to 1.4.0